.gitmodules

```diff
@@ -1,8 +1,6 @@
 [submodule "wxWidgets"]
 	path = wxWidgets
 	url = https://github.com/wxWidgets/wxWidgets
-	ignore = dirty
```

Comment:
I did not know at that, I did not think the ".gitmodule" could be used to that, I thought it was the ".gitignore" who spanked it.
But I do not understand, I deleted it, and when I actually delivered after compiling it does not appear to wxWdigets my file, so it should not be used in the end.